cmd/core-metadata/main.go

```diff
@@ -15,11 +15,15 @@ package main
 
 import (
 	"flag"
+	"github.com/edgexfoundry/edgex-go/internal/system/agent/container"
```

Comment:
These imports are in the wrong location, should be below.

Reply:
@jdharms I had made a slew of updates to `reorder` imports (across numerous `core-metadata` files) and looks like I missed this one. Thanks for catching it 👍